index.html

```diff
@@ -19,7 +19,17 @@
         box-sizing: border-box;
       }
     </style>
-    <title>Kong Spec Editor</title>
+    <title>API Documentation Editor | Kong</title>
+    <meta name="description" content="Edit, preview, and publish OpenAPI specs effortlessly with Kong's powerful API documentation editor. Designed for developers and teams who value speed and clarity.">
+    <meta property="og:title" content="API Documentation Editor | Kong" />
+    <meta property="og:description" content="Edit, preview, and publish OpenAPI specs effortlessly with Kong's powerful API documentation editor. Designed for developers and teams who value speed and clarity." />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://api-documentation.dev/" />
```

Comment:
suggestion: you should also add a canonical tag
```suggestion
    <meta property="og:url" content="https://api-documentation.dev/" />
    <link rel="canonical" href="https://api-documentation.dev/">
```